Shell/FOOLElimination.cpp

```diff
@@ -496,11 +496,11 @@ void FOOLElimination::process(Term* term, Context context, TermList& termResult,
         Formula* condition = process(sd->getCondition());
 
         TermList thenBranch;
-        Formula* thenBranchFormula;
+        Formula* thenBranchFormula {};
         process(*term->nthArgument(0), context, thenBranch, thenBranchFormula);
 
         TermList elseBranch;
-        Formula* elseBranchFormula;
+        Formula* elseBranchFormula {};
         process(*term->nthArgument(1), context, elseBranch, elseBranchFormula);
```

Comment:
Null out pointers that "should" be initialised by subsequent `process` calls. Little to no cost in release (either an XOR or hopefully dead-store elimination), traps if something disastrous happens.